hangman/lib/game/game.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Hangman.Game do
+  def new_game() do 
+    # function to filter alphabets
+    alphabets_only = fn x -> if Regex.match?(~r{[a-zA-Z]}, x),
```

Comment:
What's the difference between

~~~ elixir
if Regex.match?(~r{[a-zA-Z]}, x),
 do: true,
 else: false
~~~

and

~~~ elixir
Regex.match?(~r{[a-zA-Z]}, x)
~~~